autogen/oai/anthropic.py

```diff
@@ -64,14 +85,44 @@ def __init__(self, **kwargs: Any):
             api_key (str): The API key for the Anthropic API or set the `ANTHROPIC_API_KEY` environment variable.
         """
         self._api_key = kwargs.get("api_key", None)
+        self._aws_access_key = kwargs.get("aws_access_key", None)
+        self._aws_secret_key = kwargs.get("aws_secret_key", None)
+        self._aws_session_token = kwargs.get("aws_session_token", None)
+        self._aws_region = kwargs.get("aws_region", None)
```

Comment:
@makkzone please add the tests for these parameters. thanks

Reply:
Done.